tests/Makefile

```diff
@@ -1,13 +1,16 @@
 DESTBIN ?= 
 CUDA ?= /usr/local/cuda
 
+CHECK_INC  ?= $(shell pkg-config --cflags-only-I check)
```

Comment:
To help users who are not familiar with pkg-config, I would prefer to have CHECK_PATH possibly set by the users. If we detect CHECK_PATH, we use it to compose CHECK_INC and CHECK_LIBS. If not, get the flags from pkg-config.

Reply:
understood. I'll convert these assignments  into :=